models/DataObject/AbstractObject.php

```diff
@@ -1045,7 +1045,7 @@ public static function __callStatic(string $method, array $arguments)
             $offset = $arguments[2] ?? 0;
             $objectTypes = $arguments[3] ?? null;
 
-            $defaultCondition = $propertyName.' = '.Db::get()->quote($value).' ';
+            $defaultCondition = 'objects.'.$propertyName.' = '.Db::get()->quote($value).' ';
```

Comment:
Quote the column name should be enough
```suggestion
            $defaultCondition = $db->quoteIdentifier($propertyName) . ' = '. $db->quote($value) . ' ';
```

Reply:
You are right, just tested it! Will update the PR.